rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -609,47 +712,58 @@ private Stream<Arguments> expectedFederatorBtcPublicKeyArgs() {
                 Arguments.of(blockNumberFederationActivationFingerroot, hopActivations, federatorFromNewFederationBtcPublicKey)
             );
         }
-    }
 
-    @Test
-    void getActiveFederatorPublicKeyOfType() {
-        BtcECKey btcKey0 = BtcECKey.fromPublicOnly(Hex.decode("020000000000000000001111111111111111111122222222222222222222333333"));
-        ECKey rskKey0 = new ECKey();
-        ECKey mstKey0 = new ECKey();
+        @ParameterizedTest
+        @Tag("getActiveFederatorPublicKeyOfType")
+        @MethodSource("expectedFederatorPublicKeyOfTypeArgs")
+        void getActiveFederatorPublicKeyOfType_returnsExpectedFederatorPublicKeysAccordingToActivationAgeAndActivations(long currentBlock, ActivationConfig.ForBlock activations, BtcECKey expectedFederatorBtcPublicKey, ECKey expectedFederatorRskPublicKey, ECKey expectedFederatorMstPublicKey) {
```

Comment:
nit: I would make these long lines wrapped in various lines for redability purposes